### PR TITLE
feat(scripts): added docker_clone_volume script

### PR DESCRIPTION
**Pull request by serge1peshcoff**
it can be handy to have all the scripts in our repo, so I've added it here. I've used it a few times when migrating volumes, so this can also be useful for others

### CODE REVIEW

**Review by linuxbandit — COMMENTED** [review]
If you add this here, then I may as well add the backup script what do you think? see it in PR [with your comment](https://github.com/AEGEE/MyAEGEE/pull/155#discussion_r462166690)

**Comment by linuxbandit** [comment]
@serge1peshcoff since you made the PR you can't be a reviewer, but have a look at it. Script should work and I suggest we keep the part of volumes for now, replacing it with the other script you added later

**Review by serge1peshcoff — COMMENTED** [review]
Lgtm

**Comment by linuxbandit** [comment]
this is still wip and I will solve it this week

**Comment by WikiRik** [comment]
@linuxbandit don't forget to pull/rebase this with the current master. Since the original commit I've added the network module

**Review by linuxbandit — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -113,17 +114,20 @@ remove-agents:
 	docker-compose -f monitor-agents/docker/docker-compose.yml down
 
 # Backups
+backup:
+	./scripts/dump.sh postgres-core postgres-events postgres-statutory postgres-discounts
```
Comment (WikiRik) [inline Makefile:118]: You forgot about postgres-network

File: Makefile [hunk 2]
```diff
@@ -113,17 +114,20 @@ remove-agents:
...
 backup_statutory:
-	./helper.sh --execute postgres-statutory -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/statutory' --inserts > statutory.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
+	./scripts/dump.sh postgres-statutory
 
 backup_discounts:
-	./helper.sh --execute postgres-discounts -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/discounts' --inserts > discounts.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
+	./scripts/dump.sh postgres-discounts
 
 backup_network:
 	./helper.sh --execute postgres-network -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/network' --inserts > discounts.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
```
Comment (WikiRik) [inline Makefile:133]: This should be ./scripts/dump.sh postgres-network

File: scripts/restore.sh [hunk 3]
```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+# Restore an earlier backup done by dump.sh
+# Before executing make sure nobody is accessing the databases, i.e. stop all services except for the actual databases
+
+postgres_hosts=(postgres-core postgres-oms-statutory postgres-oms-events)
```
Comment (WikiRik) [inline scripts/restore.sh:6]: postgres-core postgres-statutory postgres-events posgres-discounts postgres-network

**Review by WikiRik — COMMENTED** [review]
You forgot about network, can you fix my comments?

File: Makefile (hunk 2 above)
Comment (linuxbandit) [reply Makefile:133]: Ah sorry, didn't notice this addition. Same for below! I will correct in another PR I have for the Makefile

**Pull request merged**